gef-ui/templates/application-details-comments.njk

```diff
@@ -0,0 +1,72 @@
+{% extends "index.njk" %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+
+{% block pageTitle %}
+  Submit to be checked at your bank
+{% endblock %}
+
+{% block content %}
+
+  {% if errors %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors.errorSummary,
+      attributes: {
+        'data-cy': 'error-summary'
+      }
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row" data-cy="application-submit-page">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+      <p class="govuk-!-margin-bottom-6">
```

Comment:
There is a nunjucks component for the back button which could be used here

Reply:
the design doesnt elude this link being that component ... but a good shout to check it out.

Reply:
OK not a problem, hadn't looked that deeply. The position in the doc and the wording seemed to shout for that component to be used, not to worry

Reply:
Ok, clarified the requirement and standard backLink now in place implemented with nunjucks component. Good catch 👍

Reply:
Nice one